src/coreclr/nativeaot/Runtime/Full/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ add_dependencies(Runtime.WorkstationGC aot_eventing_headers)
 add_library(Runtime.ServerGC STATIC ${COMMON_RUNTIME_SOURCES} ${FULL_RUNTIME_SOURCES} ${RUNTIME_SOURCES_ARCH_ASM} ${SERVER_GC_SOURCES} ${RUNTIME_ARCH_ASM_OBJECTS})
 add_dependencies(Runtime.ServerGC aot_eventing_headers)
 
+add_library(standalonegc-disabled STATIC ${CLRGC_DISABLED_SOURCES})
```

Comment:
Nit: CLRGC can be renamed to STANDALONEGC here and the remaining places for consistency.